.gitignore

```diff
@@ -1,3 +1,2 @@
 *.swp
 *.swo
-pkg/nvml/
```

Comment:
I think we should leave this in here. We should be explicit about when we want to update this folder, and you can do it with an explicit `git add -f pkg/nvml`

Reply:
I was just thinking that there was at least one other situation where we have forgotten to add generated files. Is there a way that we can check this "automatically"?

I don't feel too strongly about it though and I'm perfectly happy to revert this line.

Reply:
Yeah, I think that was a one-of case. In general, I picture us more often making changes where we explicitly *don't* want to commit back to `pkg` until the commit / PR where we do the release.

Reply:
> Yeah, I think that was a one-of case. In general, I picture us more often making changes where we explicitly _don't_ want to commit back to `pkg` until the commit / PR where we do the release.

OK. This PR now contains this commit, but I can drop it if required.